docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -366,6 +366,46 @@ For example, select the top 10 percent of nations by population::
         )
       );
 
+.. _sqlserver-procedure-function:
+
+``procedure(varchar) -> table``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``procedure`` function allows you to run stored procedure on the underlying
+database directly. It requires syntax native to SQL Server.
+
+.. note::
+
+    StoredProcedures which return multiple statement, with a non-select statement, with
+    output parameters, or with an If-Else branch are not supported.
```

Comment:
Wording is a bit confusing. Reword to "The ``procedure`` function does not support running StoredProcedures that return multiple statements, use a non-select statement, use output parameters, or use an If-Else branch." or similar